### ISSUE TITLE
[Category] Optifine seems to bug out either redstone or note blocks. 

### ISSUE THREAD

**Issue by Danteb132** [sp614x/optifine#6952]
## Description of Issue
As seen in the attached video, in the first clip I am using Optifine 1.19 pre7 and my note blocks are off in their timings when I move around. In the second clip, which is just base 1.19, they are completely fine regardless of what I do.
[](https://youtu.be/dflyuOfC1V8)
(Video too big for github)

## Steps to Reproduce
Set up redstone repeaters with noteblocks. Unsure if anything other than redstone causes this. It isn't shaders.

## OptiFine Version
Optifine 1.19 pre8

## Installation Method
Standalone installer. 

## F3 Debug Screenshot

Debug menu with Optifine:
![2022-07-07_03 34 39](https://user-images.githubusercontent.com/47989972/177717941-fc163e9c-264d-4592-b2a5-c8c380e2cf47.png)

Debug menu without Optifine:
![2022-07-07_03 36 19](https://user-images.githubusercontent.com/47989972/177718138-11f25f8b-a3f8-4142-8ebc-29b82a755cd5.png)

## Prior Testing
- Does this happen in Vanilla without OptiFine?
No, it doesn't.

## Additional Information

I updated my graphics drivers a few hours ago to the latest Nvidia ones when I saw the shaders were buggy.

**Comment by EAZYBLACK** [comment]
u mean pre7 not pre8

**Comment by Danteb132** [comment]
> 

yep, i caught that earlier but must’ve undone it. sorry. it’s fixed.

**Comment by sp614x** [comment]
Can't reproduce it unless MC is almost out of memory, only then it starts to skip a little.
